server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const app = express();
+
+// TODO - Change this to use environment variables
+const port = 8081;
+
+const eventTypes = [ 
+    {label : "cop"},
+    {label : "landl"},
+    {label : "learningEvents"} 
+];
+
+// Dynamic endpoint receiving a param of eventtype to test
+app.get('/events/:eventType', function (req, res) {
+    const foundEvent = eventTypes.find((event) => {
+        return event.label == req.params.eventType;
+    });
+    if (foundEvent) {
+        res.send(getData(foundEvent));
+    } else {
+        res.send([]);
+    }
+})
+
+function getData(event) {
+    return event;
+}
+
+// Else any requests not already hit will end up here for a 404
+app.get('/', function (req, res) {
+    console.log("Hit 404 API");
```

Comment:
We should never commit `console.log` or any log in general, it is ok-ish for now, but you should use it only when actively developing something.
The `we'll remove it later` approach is dangerous, sooner or later we are going to have an unwanted log somewhere